config/locales/en.yml

```diff
@@ -297,6 +298,11 @@ en:
   disk_evidence:
     header: 'Evidence supplied on disk'
     question: 'Will you be sending disk evidence for this claim?'
+  fee_calculator:
+    calculate:
+      amount_unavailable: The calculated rate is unavailable, please enter manually.
+    unit_price:
```

Comment:
Given this configuration for the locales, seems like [the call method](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2427/files#r209670597) has no actual difference between the children class and the parent class.